### ISSUE TITLE
sontek adv summary gives warnings on NA

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1386]
I'll be fixing this, so I won't bother explaining in detail. I converted warnings to errors, so this is not actually a dire problem, just something that ought to be checked. (PS. holy cow, there's a lot of zero velocities at the start of this timeseries.)

```
> options(warn=3)
> summary(read.oce("/data/archive/sleiwex/2008/moorings/m03/adv/sontek_b373h/raw/adv_sontek_b373h.adr", from=500000, to=500005))
ADV Summary
-----------

* Instrument:             adv, serial number ``B373H``
* Source filename:        ``/data/archive/sleiwex/2008/moorings/m03/adv/sontek_b373h/raw/adv_sontek_b373h.adr``
* Location:               unknown latitude ,  unknown longitude 
 Hide Traceback
 
 Rerun with Debug
 Error in min.default(c(NA_real_, NA_real_, NA_real_, NA_real_, NA_real_,  : 
  (converted from warning) no non-missing arguments to min; returning Inf 
10.
doWithOneRestart(return(expr), restart) 
9.
withOneRestart(expr, restarts[[1L]]) 
8.
withRestarts({
    .Internal(.signalCondition(simpleWarning(msg, call), msg, 
        call))
    .Internal(.dfltWarn(msg, call)) ... 
7.
.signalSimpleWarning("no non-missing arguments to min; returning Inf", 
    quote(min.default(structure(c(NA_real_, NA_real_, NA_real_, 
    NA_real_, NA_real_, NA_real_), class = c("POSIXct", "POSIXt"
    ), tzone = "UTC"), na.rm = TRUE))) 
6.
NextMethod(.Generic) 
5.
Summary.POSIXct(structure(c(NA_real_, NA_real_, NA_real_, NA_real_, 
NA_real_, NA_real_), class = c("POSIXct", "POSIXt"), tzone = "UTC"), 
    na.rm = TRUE) at AllClass.R#64
4.
.nextMethod(object = object) 
3.
callNextMethod() at adv.R#124
2.
summary(read.oce("/data/archive/sleiwex/2008/moorings/m03/adv/sontek_b373h/raw/adv_sontek_b373h.adr", 
    from = 5e+05, to = 500005)) 
1.
summary(read.oce("/data/archive/sleiwex/2008/moorings/m03/adv/sontek_b373h/raw/adv_sontek_b373h.adr", 
    from = 5e+05, to = 500005)) 
```

**Comment by dankelley** [comment]
As shown below, the problem is when the `to` arg is not equal to 1.

```R
> adp<-read.oce("/data/archive/sleiwex/2008/moorings/m03/adv/sontek_b373h/raw/adv_sontek_b373h.adr",from=10, to=15)
> adp[['time']]
[1] NA NA NA NA NA NA
> adp<-read.oce("/data/archive/sleiwex/2008/moorings/m03/adv/sontek_b373h/raw/adv_sontek_b373h.adr",from=1, to=15)
> adp[['time']]
 [1] "2008-06-25 12:00:12.750 UTC" "2008-06-25 12:00:12.916 UTC" "2008-06-25 12:00:13.083 UTC" "2008-06-25 12:00:13.250 UTC"
 [5] "2008-06-25 12:00:13.416 UTC" "2008-06-25 12:00:13.583 UTC" "2008-06-25 12:00:13.750 UTC" "2008-06-25 12:00:13.917 UTC"
 [9] "2008-06-25 12:00:14.083 UTC" "2008-06-25 12:00:14.250 UTC" "2008-06-25 12:00:14.417 UTC" "2008-06-25 12:00:14.584 UTC"
[13] "2008-06-25 12:00:14.750 UTC" "2008-06-25 12:00:14.917 UTC"
```

**Comment by dankelley** [comment]
Fixed in "develop" branch commit e31c80138e1c9c0b21f64c6a157ac69654dda5f5 (click that link for the diff, which is very brief in the code but extensive in the test suite, the latter being intended for use in testing the `ldc_sontek_adp` conversion to C++ as part of #1385)
